### PR TITLE
Adding OpenstackNovaInstanceStuck alerts

### PR DESCRIPTION
**Pull request by rajivmucheli**
@auhlig @Carthaca 

Please validate the OpenstackNovaInstanceStuck alerts.

### CODE REVIEW

**Comment by Carthaca** [comment]
👍

File: system/kube-monitoring/charts/prometheus-frontend/openstack-nova.alerts [hunk 1]
```diff
@@ -59,3 +59,87 @@ ALERT OpenstackNovaMaxDiskUsagePerc
...
+  FOR 5m
+  LABELS {
+    service = "nova",
+    severity = "info",
+    tier = "openstack",
+    dashboard = "nova-hypervisor"
+  }
+  ANNOTATIONS {
+    summary = "Openstack Nova Instance Stuck in Building state metric",
+    description = "Nova Instance Stuck in Building state over 15mins in {{ $.labels.host }}",
```
Comment (auhlig) [inline system/kube-monitoring/charts/prometheus-frontend/openstack-nova.alerts]: `$.labels.host` -> `$labels.host`

File: system/kube-monitoring/charts/prometheus-frontend/openstack-nova.alerts [hunk 2]
```diff
@@ -59,3 +59,87 @@ ALERT OpenstackNovaMaxDiskUsagePerc
...
+  FOR 5m
+  LABELS {
+    service = "nova",
+    severity = "info",
+    tier = "openstack",
+    dashboard = "nova-hypervisor"
+  }
+  ANNOTATIONS {
+    summary = "Openstack Nova Instance Stuck in Deleting state metric",
+    description = "Nova Instance Stuck in Deleting state over 15mins in {{ $.labels.host }}",
```
Comment (auhlig) [inline system/kube-monitoring/charts/prometheus-frontend/openstack-nova.alerts]: `$.labels.host` -> `$labels.host`

File: system/kube-monitoring/charts/prometheus-frontend/openstack-nova.alerts [hunk 3]
```diff
@@ -59,3 +59,87 @@ ALERT OpenstackNovaMaxDiskUsagePerc
...
+  FOR 5m
+  LABELS {
+    service = "nova",
+    severity = "info",
+    tier = "openstack",
+    dashboard = "nova-hypervisor"
+  }
+  ANNOTATIONS {
+    summary = "Openstack Nova Instance Stuck in Stopping state metric",
+    description = "Nova Instance Stuck in Stopping state over 15mins in {{ $.labels.host }}",
```
Comment (auhlig) [inline system/kube-monitoring/charts/prometheus-frontend/openstack-nova.alerts]: `$.labels.host` -> `$labels.host`

File: system/kube-monitoring/charts/prometheus-frontend/openstack-nova.alerts [hunk 4]
```diff
@@ -59,3 +59,87 @@ ALERT OpenstackNovaMaxDiskUsagePerc
...
+  FOR 5m
+  LABELS {
+    service = "nova",
+    severity = "info",
+    tier = "openstack",
+    dashboard = "nova-hypervisor"
+  }
+  ANNOTATIONS {
+    summary = "Openstack Nova Instance Stuck in Starting state metric",
+    description = "Nova Instance Stuck in Starting state over 15mins in {{ $.labels.host }}",
```
Comment (auhlig) [inline system/kube-monitoring/charts/prometheus-frontend/openstack-nova.alerts]: `$.labels.host` -> `$labels.host`

File: system/kube-monitoring/charts/prometheus-frontend/openstack-nova.alerts [hunk 5]
```diff
@@ -59,3 +59,87 @@ ALERT OpenstackNovaMaxDiskUsagePerc
...
+  FOR 5m
+  LABELS {
+    service = "nova",
+    severity = "info",
+    tier = "openstack",
+    dashboard = "nova-hypervisor"
+  }
+  ANNOTATIONS {
+    summary = "Openstack Nova Instance Stuck in Spawning state metric",
+    description = "Nova Instance Stuck in Spawning state over 15mins in {{ $.labels.host }}",
```
Comment (auhlig) [inline system/kube-monitoring/charts/prometheus-frontend/openstack-nova.alerts]: `$.labels.host` -> `$labels.host`

File: system/kube-monitoring/charts/prometheus-frontend/openstack-nova.alerts [hunk 6]
```diff
@@ -59,3 +59,87 @@ ALERT OpenstackNovaMaxDiskUsagePerc
...
+  FOR 5m
+  LABELS {
+    service = "nova",
+    severity = "info",
+    tier = "openstack",
+    dashboard = "nova-hypervisor"
+  }
+  ANNOTATIONS {
+    summary = "Openstack Nova Instance Stuck in Rebooting state metric",
+    description = "Nova Instance Stuck in Rebooting state over 15mins in {{ $.labels.host }}",
```
Comment (auhlig) [inline system/kube-monitoring/charts/prometheus-frontend/openstack-nova.alerts]: `$.labels.host` -> `$labels.host`

**Pull request merged**